packages/story-editor/src/utils/storyPageToNode.js

```diff
@@ -92,13 +92,15 @@ const PageWithDependencies = forwardRef(function PageWithDependencies(
                     renderFullHeightThumb ? (containerHeight - height) / 2 : 0
                   }
                 >
-                  {page.elements.map((element) => (
-                    <DisplayElement
-                      key={element.id}
-                      previewMode
-                      element={element}
-                    />
-                  ))}
+                  {page.elements
+                    .filter((element) => !element.isHidden)
+                    .map((element) => (
+                      <DisplayElement
```

Comment:
Maybe it would be simpler to just add this to the `DisplayElement` directly so anywhere its used it'll be hidden?

Reply:
👍🏻 , changed.